### PR TITLE
[LPN-1345] add department sites list controller

### PR DESCRIPTION
**Pull request by elvino29**

### CODE REVIEW

File: src/Repository/Geo/ZoneRepository.php [hunk 1]
```diff
@@ -500,4 +514,19 @@ public function searchCitiesInZones(array $zones, string $search): array
 
         return $qb->getQuery()->getResult();
     }
+
+    public function findAllDepartmentSiteIndexByCode(): array
```
Comment (ottaviano) [inline src/Repository/Geo/ZoneRepository.php:518]: what about moving this into DepartmentRepository?

File: src/Controller/Renaissance/DepartmentSite/DepartmentSiteController.php [hunk 2]
```diff
@@ -13,6 +15,20 @@
...
 class DepartmentSiteController extends AbstractController
 {
+    use CanaryControllerTrait;
+
+    /**
+     * @Route(name="list", methods={"GET"})
+     */
+    public function departmentSiteListAction(ZoneRepository $zoneRepository): Response
+    {
+        $this->disableInProduction();
```
[the remark this answers is not in the document]
Comment (elvino29) [reply src/Controller/Renaissance/DepartmentSite/DepartmentSiteController.php:25]: No more

File: src/Repository/Geo/ZoneRepository.php (hunk 1 above)
Comment (elvino29) [reply src/Repository/Geo/ZoneRepository.php:548]: we don't have a custom department like 64PB or FDE in the departmental table

File: src/Repository/Geo/ZoneRepository.php [hunk 3]
```diff
@@ -530,4 +544,19 @@ public function findOneByCode(string $code): ?Zone
...
+    {
+        return $this->createQueryBuilder('zone', 'zone.code')
+            ->select('zone.name', 'zone.code')
+            ->addSelect('site.slug AS site_slug')
+            ->leftJoin(DepartmentSite::class, 'site', Join::WITH, 'zone = site.zone')
+            ->where('zone.type = :dpt')
+            ->orderBy('zone.name', 'ASC')
+            ->setParameter('dpt', Zone::DEPARTMENT)
+            ->getQuery()
+            ->enableResultCache(2678400) // 31 days
```
[the remark this answers is not in the document]
Comment (elvino29) [reply src/Repository/Geo/ZoneRepository.php:558]: Yes, of course. I'll remove it

**Review by ottaviano — APPROVED** [review]

**Pull request merged**